spec/models/history_entry_spec.rb

```diff
@@ -129,6 +129,14 @@
       s._id.in?(res).should be_falsy
     end
 
+    it "should get value at limit dates" do
+      s = Contact.make(level: 'aspirante')
+      s.history_entries.delete_all
+      s.history_entries.create(attribute: 'level', old_value: nil, changed_at: "2015-10-31 14:35".to_time(:utc))
```

Comment:
Line is too long. [112/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.